sources/packages/forms/src/form-definitions/institutionprofile.json

```diff
@@ -346,7 +346,8 @@
           "addons": [],
           "inputType": "text",
           "id": "eiloy9e",
-          "defaultValue": ""
+          "defaultValue": "",
+          "isNew": false
```

Comment:
just wondering, are we not adding the max length to legalOperatingName

Reply:
We consider that @andrewsignori-aot but the `legalOperatingName` is displayed here for info only. The value is retrieved from the token directly, not from the UI, so there is no real point in adding a validation here since we have the confirmation about the 250 characters.

Reply:
Yes, the value is read from the response of bceid web service. 

![image](https://github.com/bcgov/SIMS/assets/54600590/f5744c81-eee1-4369-a644-0ac27f439e48)

Reply:
@andrewsignori-aot , I just thought twice, we are re-using the same form(institution profile) to create institution by ministry right. should we use the validation then?

![image](https://github.com/bcgov/SIMS/assets/54600590/fac44a83-295b-4e25-ad6b-3633812cdc64)

Reply:
mm, ya...  but I was wondering, what if they increased the length (and we have 250), and now there is a new institution with more than 250 characters of legal name, then the same error which we saw today will happen again right? but if we add the max length here, at least the user will not be able to submit due to validation and the user will at least know the issue right?

but, I am good with what we have in the PR, just sharing my thoughts

Reply:
Yes, that makes sense @dheepak-aot, nice catch.